haskell/test-common/Lorentz/Contracts/Test/Common.hs

```diff
@@ -224,3 +225,6 @@ mkInitialStorage OriginationParams{..} = let
       -> Map (Address, (Address, Natural)) ()
       -> Map (Address, (Address, Natural)) ()
     foldFn ow ops m = foldr (\a b -> Map.insert (ow, (a, 0)) () b) m ops
+
+mgmContractPaused :: ExecutorError -> IntegrationalScenario
+mgmContractPaused = lExpectFailWith (== [mt|CONTRACT_PAUSED|])
```

Comment:
Why did you move it btw?

Reply:
@gromakovsky To use in the `Lorentz.Contracts.Test.FA2` module to test that `update_operator` is pausable.

Reply:
Ah, I thought it previously was in the library. Ok then.